collect_app/src/main/java/org/odk/collect/android/adapters/InstanceUploaderAdapter.java

```diff
@@ -17,24 +17,13 @@
 
 import java.util.Date;
 
-import io.reactivex.disposables.CompositeDisposable;
-
 import static org.odk.collect.forms.instances.Instance.STATUS_SUBMISSION_FAILED;
 import static org.odk.collect.forms.instances.Instance.STATUS_SUBMITTED;
 
 public class InstanceUploaderAdapter extends CursorAdapter {
-    private final CompositeDisposable compositeDisposable;
```

Comment:
Was this just here and not being used for anything?

Reply:
Yes